@lingosta/app/components/Translate/LangListbox.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
```

Comment:
This file is great. Don't worry about using it for now. I'm going to incorporate it to `groups` so that languages are selected at the group-level, not for individual translations.